ebu_tt_live/documents/converters.py

```diff
@@ -30,3 +32,16 @@ def ebutt3_to_ebuttd(ebutt3_in, media_clock):
     ebuttd_document = EBUTTDDocument.create_from_raw_binding(ebuttd_bindings)
     ebuttd_document.validate()
     return ebuttd_document
+
+
+def ebutt1_to_ebutt3(ebutt1_in):
+    """
+    This function takes an EBUTT1Document instance and returns the same document as an EBUTT3Document instance.
+    :param ebutt1_in:
+    :return:
+    """
+    converter = EBUTT1EBUTT3Converter()
+    ebutt3_bindings = converter.convert_document(ebutt1_in.binding)
+    ebutt3_document = EBUTT3Document.create_from_raw_binding(ebutt3_bindings)
```

Comment:
I'm puzzled about where the sequence identifier is added - there doesn't seem to be any data available to the converter to allow it to assign one. Without it, the `validate()` step will fail. If one is being assigned, the value needs to come from somewhere.

Reply:
Decided we shouldn't do this now, in this pull request.